### PR TITLE
Sort

### PR DESCRIPTION
**Pull request by ITarako**
i will write e2e-tests tomorrow, because i don't want die, while i will try install webdriver on notebook

### CODE REVIEW

**Comment by Eoksni** [comment]
I guess its ready now? For future, please never-EVER push without at least **some** manual testing. The very basic "happy path" must be successful.

**Comment by ITarako** [comment]
Before push, all is work. But..I decided rename prop of vue component . It's EZ, only EZ rename, no change in functionality or something else, so after rename I push it and in result I have very stypid error

**Comment by Eoksni** [comment]
No worries, happens to everyone - renaming in js is hard due to its dynamic nature. Just be more careful next time, and don't forget about "Find and replace in project" Ctrl-Shift-H feature :)
I'll review it tomorrow morning, don't have time atm.

File: config.json.example [hunk 1]
```diff
@@ -1,6 +0,0 @@
-{
-    "default_language": "en",
-    "db": "mongo",
-    "db_url": "mongodb://localhost:27017/kladovka",
-    "express_port": 8080
-}
```
Comment (Eoksni) [inline config.json.example]: why did you remove config.json.example?

File: frontend/src/components/kl-items.js [hunk 2]
```diff
@@ -45,6 +52,9 @@ export default {
                 item.totalDps = totalDps(item, newVal).toFixed(2);
                 return item;
             });
+        },
+        sort: function (newVal) {
+            this[newVal]();
         }
```
Comment (Eoksni) [inline frontend/src/components/kl-items.js]: List will become unsorted if you add another item after sorting, won't it?

The proper way is like so:

- Have a value to indicate which way to sort (you already have it, `sort` it is)
- Have a [computed property](https://vuejs.org/v2/guide/computed.html) `sorted` which is, well, a sorted list of items, created using `items` and `sort`
- Bind your list in template to `sorted` instead of `items`

File: frontend/src/components/kl-items.vue [hunk 3]
```diff
@@ -23,7 +34,7 @@
                 </div>
             </li>
         </ul>
-        <div class="manipulate-all">
+        <div class="manipulate-all">
```
Comment (Eoksni) [inline frontend/src/components/kl-items.vue]: Awesome change!

File: config.json.example (hunk 1 above)
Comment (ITarako) [inline config.json.example]: ooops

File: frontend/src/components/kl-items.vue (hunk 3 above)
Comment (ITarako) [inline frontend/src/components/kl-items.vue]: I installed extension of vscode for this
